lib/matplotlib/cbook.py

```diff
@@ -2445,6 +2445,14 @@ def safe_first_element(obj):
     return next(iter(obj))
 
 
+def asSequence(data):
```

Comment:
Can you call this something like `view_as_sequence`?  named as it is it is a bit ambiguous as to what it does (and more importantly does not do).